modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/DatabaseReaderLazyLoader.java

```diff
@@ -194,6 +196,9 @@ int current() {
     }
 
     DatabaseReader get() throws IOException {
+        if (lastUpdate != 0 && System.currentTimeMillis() - lastUpdate > Duration.ofDays(30).toMillis()) {
+            throw new IllegalStateException("Database was not updated for 30 days");
```

Comment:
I think we should only check this for databases downloaded by the registry.
(This check now also applies for default and config databases?)

Maybe a flag here that indicates that this db originates from geoip database index?

Reply:
`lastUpdate != 0` marks downloaded databases, we never update it for default/config databases, I think this is enough, WDYT?

Reply:
Yes, I missed that, this makes sense.